cmd/kubeadm/.import-restrictions

```diff
@@ -120,6 +120,7 @@
 				"k8s.io/kubernetes/pkg/scheduler/algorithm",
 				"k8s.io/kubernetes/pkg/scheduler/api",
 				"k8s.io/kubernetes/pkg/scheduler/cache",
+				"k8s.io/kubernetes/pkg/scheduler/internal/cache",
```

Comment:
golang will not allow imports of `pkg/scheduler/internal` from outside `pkg/scheduler`. I think this line should be removed. (same for kubectl.)

Reply:
if remove it, some error will like:

errors in package "k8s.io/kubernetes/pkg/kubectl/cmd/testing":
import k8s.io/kubernetes/pkg/scheduler/internal/cache did not match any allowed prefix.

Reply:
OK - I'm fine leaving it as-is. The go compiler should enforce our intentions anyway.